### PR TITLE
[thci] update `pollperiod` command to accept ms resolution configuration

### PR DESCRIPTION
**Pull request by FiveDimensions**
After convert s to ms, here is the cmd log

pollperiod
236000
Done
pollperiod 500
Done
pollperiod
500
Done
pollperiod 1
Done
pollperiod
1
Done

### CODE REVIEW

File: src/cli/cli.cpp [hunk 1]
```diff
@@ -2001,12 +2001,12 @@ void Interpreter::ProcessPollPeriod(int argc, char *argv[])
...
     if (argc == 0)
     {
-        mServer->OutputFormat("%d\r\n", (otLinkGetPollPeriod(mInstance) / 1000)); // ms->s
+        mServer->OutputFormat("%d\r\n", otLinkGetPollPeriod(mInstance));
     }
     else
     {
         SuccessOrExit(error = ParseLong(argv[0], value));
-        otLinkSetPollPeriod(mInstance, static_cast<uint32_t>(value * 1000)); // s->ms
+        otLinkSetPollPeriod(mInstance, static_cast<uint32_t>(value));
```
Comment (librasungirl) [inline src/cli/cli.cpp]: Allowed minimal data poll rate should be 10ms (OPENTHREAD_CONFIG_MINIMUM_POLL_PERIOD).

How about adding check for the value and default as 10ms for not allowed values?
https://github.com/openthread/openthread/blob/master/src/core/thread/data_poll_manager.cpp#L166

**Comment by FiveDimensions** [comment]
pollperiod >= 10ms:
> pollperiod
236000
Done
> pollperiod 500
Done
> pollperiod
500
Done
> pollperiod 10
Done
> pollperiod
10
Done
> pollperiod 9
Error 6: Parse
> pollperiod 0
Error 6: Parse
> pollperiod -1
Error 6: Parse
> pollperiod
10
Done

File: src/cli/cli.cpp [hunk 2]
```diff
@@ -2001,12 +2001,13 @@ void Interpreter::ProcessPollPeriod(int argc, char *argv[])
...
     if (argc == 0)
     {
-        mServer->OutputFormat("%d\r\n", (otLinkGetPollPeriod(mInstance) / 1000)); // ms->s
+        mServer->OutputFormat("%d\r\n", otLinkGetPollPeriod(mInstance));
     }
     else
     {
         SuccessOrExit(error = ParseLong(argv[0], value));
-        otLinkSetPollPeriod(mInstance, static_cast<uint32_t>(value * 1000)); // s->ms
+        VerifyOrExit(!(value < OPENTHREAD_CONFIG_MINIMUM_POLL_PERIOD), error = OT_ERROR_PARSE);
```
Comment (bukepo) [inline src/cli/cli.cpp]: => `VerifyOrExit(value >= OPENTHREAD_CONFIG_MINIMUM_POLL_PERIOD, error = OT_ERROR_INVALID_ARGS);`

File: tools/harness-thci/OpenThread.py [hunk 3]
```diff
@@ -1312,9 +1318,11 @@ def setPollingRate(self, iPollingRate):
             False: fail to set the data polling rate for sleepy end device
         """
         print '%s call setPollingRate' % self.port
-        print iPollingRate
+        # convert s to ms
+        iPollingRate *= 1000
+        print int(iPollingRate)
```
Comment (bukepo) [inline tools/harness-thci/OpenThread.py]: use `print` style compatible with python3?

Comment (FiveDimensions) [reply tools/harness-thci/OpenThread.py]: @bukepo Currently THCI uses python 2.7 and there are a lot of python2-style statements in THCI files. If needed, we should change them all in the future.

**Comment by jwhui** [comment]
@FiveDimensions, can you address/respond to the open comments above?

File: src/cli/cli.cpp (hunk 1 above)
Comment (FiveDimensions) [reply src/cli/cli.cpp]: Ok, updated in the second commit

File: src/cli/cli.cpp (hunk 2 above)
Comment (FiveDimensions) [reply src/cli/cli.cpp]: Ok, updated in the third commit

File: src/cli/cli.cpp (hunk 1 above)
Comment (jwhui) [reply src/cli/cli.cpp]: @librasungirl, where were you suggesting to add this check?

Comment (librasungirl) [reply src/cli/cli.cpp]: planed to submit a separate PR to enhance this check.
Just submitted #3401, please have a review.

**Pull request merged**